src/modules/Dashboard/features/Participants/Participants.utils.tsx

```diff
@@ -131,16 +130,9 @@ export const getHeadCells = (id?: string): HeadCell[] => {
   const { t } = i18n;
 
   return [
-    {
-      id: 'checkbox',
-      label: <Checkbox aria-label={t('checkAll')} checked={false} />,
-      enableSort: true,
-      width: ParticipantsColumnsWidth.Pin,
-    },
     {
       id: 'pin',
       label: '',
-      enableSort: true,
```

Comment:
I also removed the "Sort" button from the Pin column on the Participants table, since that appeared to defeat the purpose of pinning, and isn't included in the list of sortable columns in Figma.